keel-igor/src/main/kotlin/com/netflix/spinnaker/config/IgorConfiguration.kt

```diff
@@ -13,7 +13,7 @@ import retrofit2.Retrofit
 import retrofit2.converter.jackson.JacksonConverterFactory
 
 @Configuration
-@ConditionalOnProperty("igor.enabled")
+@ConditionalOnProperty("igor.base-url")
```

Comment:
I propose we just get rid of this and make keel expect that an Iger is available